src/main/java/emu/grasscutter/Grasscutter.java

```diff
@@ -154,6 +154,7 @@ public static void loadLanguage() {
 			Grasscutter.getLogger().error("Illegal locale language, using en-US instead.");
 			language = Language.getLanguage("en-US");
 		} else {
+			languageTag = String.format("%s-%s", locale.getCountry(), locale.getLanguage());
```

Comment:
`country`-`language` ?
`CN`-`zh` ?

Reply:
Thanks for your review, this bug has been fixed.